students/jarosz_natalia/lesson_08/regex_version_of strip.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+
+def regex_strip(string, chars_to_remove):
+    strip_regex = re.compile('[%s]' % chars_to_remove)
```

Comment:
Your regex should be a bit rebuilt. Lets consider something like this:
```re.compile("^([" + chars_to_remove+ "]*) | ([" + chars_to_remove+ "]*)$")```

With this small change behaviour of your regex_strip function will be the same as strip() function